modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -47,6 +47,28 @@ In the following example, the app will perform an `ADD_TASK` mutation which will
 
 For more information, see link:https://www.apollographql.com/docs/react/essentials/mutations.html#props[Apollo's document about mutations].
 
+== Offline Workflow
+
+When offline `client.mutate` function will return immediately after is called.
+Returned promise will resolve into error (`catch` method is triggered).
+Developers can detect if error is an offline error and watch for change to be replicated back to server.
+
+For example: 
+[source, javascript]
+----
+  client.mutate(...).catch((error)=> {
+    // 1. Detect if this was an offline error
+   if(error.networkError && error.networkError.offline){
+     const offlineError: OfflineError =  error.networkError;
+     // 2. We can still track when offline change is going to be replicated.
+     offlineError.watchOfflineChange().then(...)
+   }
+  });
+----
+
+Note: Additionally to watching individual mutations framework offers global offline listener that can be supplied when creating client.
```

Comment:
```suggestion
NOTE:  In addition to watching individual mutations, you can add a global offline listener when creating a client.
```
do we provide docs for that?

Reply:
Yes. It `Listening to changes` chapter bellow